messages/index.js

```diff
@@ -525,6 +544,7 @@ module.exports = {
     cs: 'musí vyhovět právě jednomu schématu v "oneOf"',
     de: 'muss genau einem der Schemata in "oneOf" entsprechen',
     es: 'debe coincidir con un solo esquema en "oneOf"',
+    fi: 'täytyy vastata "oneOf" -kohdassa määriteltyä skeemaa',
```

Comment:
I think "exactly one" could be missing here?

Reply:
Added statement for "exactly one".